.github/workflows/make_release.yml

```diff
@@ -59,13 +59,15 @@ jobs:
         if: ${{ steps.check_already_built.outputs.should_build == '1' }}
         # 1) Setup in-docker auth against gcloud
         # 2) Remove unnecessary debug build
+        # 3) Remove unnecessary Wasm build and publish
         # Note that /work is the mount destination of the current home of make command with BUILD_WITH_CONTAINER=1.
         run: |
           CREDENTIAL_FILE_NAME=$(echo ${GOOGLE_APPLICATION_CREDENTIALS} | awk -F/ '{print $NF}')
           cp scripts/release-binary.sh scripts/release-binary.sh.tmp
           echo gcloud auth activate-service-account --key-file="/work/${CREDENTIAL_FILE_NAME}" > scripts/release-binary.sh
           cat scripts/release-binary.sh.tmp >> scripts/release-binary.sh
           sed -i 's/release release\-symbol debug/release release\-symbol/' scripts/release-binary.sh
+          sed -i -n '/Build and publish Wasm plugins/q;p' scripts/release-binary.sh
```

Comment:
Is the approach safe just in case release-binary upstream script changes and additional code added to the end?
Can we leverage this variable - ${BUILD_ENVOY_BINARY_ONLY}?

A naive question - these WASMs are used to create istio proxyv2 containers. And the names of the WASM has the SHA in it? If the WASMs are not available, wont proxyv2 creation fail? I can be wrong.